src/tsickle.ts

```diff
@@ -2017,6 +2025,51 @@ export function emitWithTsickle(
   };
 }
 
+function addClutzAliases(
+    dtsFileContent: string, sourceFiles: ts.SourceFile[], typeChecker: ts.TypeChecker,
+    host: TsickleHost): string {
+  let reexports = '';
+  for (const sf of sourceFiles) {
+    const moduleSymbol = typeChecker.getSymbolAtLocation(sf);
+    // const moduleExports = moduleSymbol && moduleSymbol.exports;
+    const moduleExports = moduleSymbol && typeChecker.getExportsOfModule(moduleSymbol);
+
+    if (!moduleExports) {
+      return dtsFileContent;
+    }
+
+    // moduleExports is a ts.Map<ts.Symbol> which is an es6 Map, but has a
+    // different type for no reason
+    for (const symbol of moduleExports) {
+      // Want to alias the symbol to match what clutz would produce
+      // see examples at
+      // https://github.com/angular/clutz/tree/master/src/test/java/com/google/javascript/clutz
```

Comment:
Is this load bearing, or just something we do to mimick Clutz? Can you comment on that?